resources/migrations/049_add_pulse_skip_if_empty.yaml

```diff
@@ -0,0 +1,15 @@
+databaseChangeLog:
+  - changeSet:
+      id: 49
+      author: tlrobinson
+      changes:
+        ######################################## add skip_if_empty column to pulse ########################################
```

Comment:
no real need for this comment here, there's only one thing going on in this migration so no need to break it into sections